package.json

```diff
@@ -92,6 +92,7 @@
     "minimist": "^1.2.0",
     "mongoose": "^4.9.8",
     "morgan": "^1.8.1",
+    "ms": "^2.0.0",
```

Comment:
We can either preparse the duration using `parse-duration` before passing it to the JWT Token Generator. Or port our codebase to `ms`.